src/admin/index.spec.js

```diff
@@ -79,15 +81,20 @@ describe('Admin', () => {
     const requestListener = jest.fn().mockName('request_timeout')
     admin.on(admin.events.REQUEST_TIMEOUT, requestListener)
 
-    await admin
-      .connect()
-      .then(() =>
-        admin.createTopics({
+    await admin.connect()
+
+    await retrier(async () => {
+      try {
+        await admin.createTopics({
           waitForLeaders: false,
           topics: [{ topic: `test-topic-${secureRandom()}` }],
         })
-      )
-      .catch(e => e)
+      } catch (e) {
+        if (e.name !== 'KafkaJSRequestTimeoutError') {
```

Comment:
The theory here is that in some circumstances, `createTopics` failed with some other error than a request timeout, which would make the test fail. Now we retry on any other retriable failure, which should hopefully fix it.